src/private/utils.js

```diff
@@ -62,13 +62,10 @@ const getOwnKeys = getOwnPropertySymbols
 
 
 export function getOwnPropertyDescriptors(obj) {
-  const descs = {};
-
-  getOwnKeys(obj).forEach(
-    key => (descs[key] = getOwnPropertyDescriptor(obj, key))
-  );
-
-  return descs;
+  return getOwnKeys(obj).map(key => [
```

Comment:
Perhaps instead of using `getOwnPropertyDescriptors` (pural) for this task, instead we do something like:
```js
const keys = getOwnPropertyNames(klass.prototype);
for (const key of keys) {
  // singular
  const desc = getOwnPropertyDescriptor(klass.prototype, key);
  
  // etc
  defineProperty(klass.prototype, key, autobindMethod(klass.prototype, key, desc));
}
```